packages/react/src/stories/ic-theme.stories.mdx

```diff
@@ -185,3 +189,197 @@ export const Controlled = () => {
     </div>
   </Story>
 </Canvas>
+
+
+### Playground
+
+export const defaultArgs = {
+  color: "rgba(27, 60, 121, 1)",
```

Comment:
Do we need to set a color?

Reply:
it's needed to set the colour of the top nav\hero etc (the prop will be renamed "brand" in a future ticket)